js/audio/shared/VolumeControl.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+	import { onMount } from "svelte";
+	import WaveSurfer from "wavesurfer.js";
+
+	export let currentVolume = 1;
+	export let show_volume_slider = false;
+	export let waveform: WaveSurfer;
+
+	onMount(() => {
+		adjustSlider();
+	});
+
+	const adjustSlider = (): void => {
+		let slider = document.getElementById("volume");
```

Comment:
won't this approach only allow one audio element at a time? we should be binding elements and using references to them: https://svelte.dev/repl/76a65cdbd14f40a5818ab327c7b3105b?version=3.29.4